src/Admin/Ajax.php

```diff
@@ -551,11 +551,11 @@ private function optionUnnavailableInTestMode(){
 	public function payplug_save_data( WP_REST_Request $request ) {
 
 		$payplug = new PayplugGateway();
-
+		$options = PayplugWoocommerceHelper::get_payplug_options();
```

Comment:
Can we use the `$payplug->settings` ? seems more performant

Reply:
this will need a new PayplugGateway everytime we are calling the method

Reply:
but it's already being declared right on this line (553), right?